style/index.css

```diff
@@ -3,8 +3,99 @@
 | Distributed under the terms of the Modified BSD License.
 |----------------------------------------------------------------------------*/
 
-.jp-DebuggerSidebar {
+.jp-Debugger {
   background: var(--jp-layout-color1);
   top: 0;
   bottom: 0;
+}
+
+
+.jp-DebuggerSidebar {
+  display: flex;
+  flex-direction: column;
+  color: var(--jp-ui-font-color1);
+  background: var(--jp-layout-color1);
+  font-size: var(--jp-ui-font-size1);
+  min-width: 350px !important;
+}
+
+
+
+
+.jp-DebuggerSidebar  div:not(:first-child) .debugger-variables__header{
+  border-bottom: solid var(--jp-border-width) var(--jp-border-color1);
+  border-top: solid var(--jp-border-width) var(--jp-border-color1);
+}
+
+.debugger-variables__header {
```

Comment:
For the CSS naming conventions, I would propose giving *each* class its own top level name. If they are semantically children of something (for example the variables widget will always live inside a `DebuggerSidebar` then it should look something like this: `.jp-DebuggerSidebar-variables`.

Reply:
Or even `.jp-DebuggerVariables` and that way the sub-item could be like `jp-DebuggerVariables-header` etc.

Reply:
Ok, cool. I will refactor all css class in index.css